test/rdcpu_sanitytest.c

```diff
@@ -137,7 +137,11 @@ int setup_tests(void)
         ADD_TEST(sanity_check_rdseed_bytes);
     }
 # elif IS_AARCH_64
+#  if defined(OPENSSL_NO_ASM)
```

Comment:
I doubt this is the right approach.
I think the `OPENSSL_cpuid_setup()` above will not be available when
`OPENSSL_CPUID_OBJ` is not defined.